_shows/06_07/my_fair_lady.md

```diff
@@ -8,6 +8,28 @@ season_sort: 220
 venue: New Theatre
 date_start: 2007-02-14
 date_end: 2007-02-17
+
+cast:
+  - role: Henry Higgins
+    name: Peter Carruthers
+  - role: Colonel Hugh Pickering
+    name: Guy Unsworth
+  - role: Eliza Doolittle
+    name: Melanie Birkhead
+  - role: Freddie Eynsford-Hill
+    name: Chris McDonald
+  - role: Alfie Doolittle
+    nane: James Rowe
```

Comment:
James Rowe has not rendered on the netlify build. Any ideas? Just got 'Alfie Doolittle' with nothing underneath.